features/step_definitions/assertion_steps.rb

```diff
@@ -52,9 +52,10 @@ def get_path(pagename, user = @user)
         path = new_shf_document_path
       when 'user details'
         path = user_path(user)
-      else
-        fail("no path defined for \"#{pagename}\"")
     end
+
+    expect(path).not_to be_empty, "A step was called with path= /'#{pagename}/', but that path is not defined in #{__method__}"
+
```

Comment:
Do you have the backslashes in the output string on purpose? Or should they be escapes?

`expect(path).not_to be_empty, "A step was called with path=\"#{pagename}\", but that path is not defined in #{__method__}"`